calico_kubernetes/calico_kubernetes.py

```diff
@@ -40,6 +42,11 @@
 # Allow the user to enable/disable namespace isolation policy
 DEFAULT_POLICY = os.environ.get('DEFAULT_POLICY', 'allow')
 
+# Flag to indicate whether or not to use Calico IPAM
```

Comment:
Make sure to use proper punctuation on your comments! That means periods at the end!